### PR TITLE
Orange-OpenSource.Hurl version 2.0.1

### PR DESCRIPTION
**Pull request by lepapareil**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95399)

### CODE REVIEW

**Comment by lepapareil** [comment]
Hi @stephengillie,

I saw that you've been the one that approved our last [MR for Hurl 2.0.0](https://github.com/microsoft/winget-pkgs/pull/94632), can you help me by approving this one as it is opened since feb 2 please 😄  ?

**Comment by stephengillie** [comment]
Hi @lepapareil,

This version installs normally, but requires `Microsoft.VCRedist.2015+.x64` to run:
image

**Comment by lepapareil** [comment]
> Hi @lepapareil,
> 
> This version installs normally, but requires `Microsoft.VCRedist.2015+.x64` to run: image

Thks @stephengillie.

I don't understand what's happening for now because there is no fundamental change between 2.0.0 and 2.0.1 (just a bug fix). I will work on it.

**Comment by lepapareil** [comment]
Hi @stephengillie,

I just tested our package on two clean `windows 10 HOME` and `windows 10 PRO` with the officials Microsoft isos and `vcruntime140.dll` exists by default on `C:\system32`, that's why we don't embed it in our package (like CRYPT32.dll, kernel32.dll, WS2_32.dll and bcrypt.dll). What do you suggest to make this version valid for winget ?

File: manifests/o/Orange-OpenSource/Hurl/2.0.1/Orange-OpenSource.Hurl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Orange-OpenSource.Hurl
+PackageVersion: 2.0.1
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+Installers:
```
Comment (stephengillie) [inline manifests/o/Orange-OpenSource/Hurl/2.0.1/Orange-OpenSource.Hurl.installer.yaml:8]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/o/Orange-OpenSource/Hurl/2.0.1/Orange-OpenSource.Hurl.installer.yaml:8]: Hi @lepapareil,

Sorry to forget the label yesterday, as the accompanying message is somewhat informative and may have answered your question. The root issue is that Microsoft.VCRedist.2015+.x64 is required for this package to execute, but is not included on all systems. Our validations are performed on fresh installs of Windows, and ours do not have these files preinstalled.

Once dependencies are fully supported, this suggestion should have WinGet install this dependency before installing the Orange main package.

Comment (lepapareil) [reply manifests/o/Orange-OpenSource/Hurl/2.0.1/Orange-OpenSource.Hurl.installer.yaml:11]: Many thks @stephengillie :)

I pushed your suggestion https://github.com/lepapareil/winget-pkgs/commit/441e052c1b2307f7bab66a756f3b985ca6dd4131

**Comment by lepapareil** [comment]
Hi @stephengillie,

Do I have to do anything else to validate this version plz ?

**Comment by stephengillie** [comment]
In a Windows 11 VM, this installs and runs without issue. It's expecting arguments:
```
Run Hurl file(s) or standard input

Usage: hurl [OPTIONS] [FILE]...

Arguments:
  [FILE]...  Sets the input file to use

Options:
      --cacert <FILE>                         CA certificate to verify peer against (PEM format)
  -E, --cert <CERTIFICATE[:PASSWORD]>         Client certificate file and password
      --key <KEY>                             Private key file name
...
```

**Review by stephengillie — APPROVED** [review]

**Comment by lepapareil** [comment]
thks for your work @stephengillie  :)

**Pull request merged**